.github/workflows/continuous-deployment.yml

```
@@ -1,45 +1,54 @@
 name: Continuous Deployment
 
 on:
+  on:
```

actionlint (1.7.7)

unknown Webhook event "on". see https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#webhook-events for list of all Webhook event names

(events)